docs/pr-reviews.md

```diff
@@ -0,0 +1,20 @@
+# PR Reviews
+
```

Comment:
Before PR section? We can add links to github-pr-labels, systemd-rules and tox testing. We are assuming they know all that when they are reading the rest